.github/workflows/python-cicd-units.yaml

```diff
@@ -0,0 +1,21 @@
+name: Python CICD pipelines - Unittesting
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install boto3 click hatanaka matplotlib numpy pandas plotext==4.2 plotly pymongo pytest scipy tqdm unlzw3 typing_extensions
```

Comment:
This duplicates specification of our requirements. It's not as simple as just pointing at `requirements.txt` though, as there isn't one.

Requirements are specified in `setup.py`, though that _could_ be modified to read them from a `requirements.txt` file for consistency... I prototyped that idea and it works, but it's hacky because I have to handle possible commenting. Time for me to read about the differences between setup.py and requirements.txt and how to do this properly...